openfl-workspace/default/shard_config.yaml

```diff
@@ -0,0 +1,5 @@
+template: shard_descriptor.LocalShardDescriptor
+params:
+  data_path: data
+  sample_shape: []
```

Comment:
Is the idea here that `shard_config.yaml` and `shard_descriptor.py` get copied into the default workspace? I would request that a short comment is provided here that explains each of the parameters that need to be filled in by the users